### PR TITLE
Split lambda_event_source_mapping_supplier

### LINKED ISSUE #733: Split lambda_event_source_mapping_supplier

### PR DESCRIPTION
**Pull request by moadibfr**
fixes #733

### CODE REVIEW

File: pkg/remote/aws/init.go [hunk 1]
```diff
@@ -137,8 +137,12 @@ func Init(version string, alerter *alerter.Alerter,
 	remoteLibrary.AddEnumerator(NewDynamoDBTableEnumerator(dynamoDBRepository, factory))
 	remoteLibrary.AddDetailsFetcher(aws.AwsDynamodbTableResourceType, NewDynamoDBTableDetailsFetcher(provider, deserializer))
 
+	remoteLibrary.AddEnumerator(NewLambdaFunctionEnumerator(lambdaRepository, factory))
+	remoteLibrary.AddDetailsFetcher(aws.AwsLambdaFunctionResourceType, NewLambdaFunctionDetailsFetcher(provider, deserializer))
+	remoteLibrary.AddEnumerator(NewLambdaEventSourceMappingEnumerator(lambdaRepository, factory))
+	remoteLibrary.AddDetailsFetcher(aws.AwsLambdaEventSourceMappingResourceType, common.NewGenericDetailsFetcher(aws.AwsSqsQueuePolicyResourceType, provider, deserializer))
```
Comment (sundowndev) [inline pkg/remote/aws/init.go:143]: wrong type passed to `NewGenericDetailsFetcher`

**Review by sundowndev — CHANGES_REQUESTED** [review]

File: pkg/remote/aws/init.go (hunk 1 above)
Comment (moadibfr) [reply pkg/remote/aws/init.go]: no it was rebased over lambda function before I broke the pr -_-

Comment (moadibfr) [reply pkg/remote/aws/init.go:150]: ha sorri I missread the comment :O

Comment (moadibfr) [reply pkg/remote/aws/init.go:150]: ha sorri I missread the comment :O

**Review by sundowndev — APPROVED** [review]

**Review by wbeuil — APPROVED** [review]

**Pull request merged**